.github/workflows/publish-internal-prod.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and Deploy to NR Internal Prod
+
+on:
+  workflow_dispatch:
+    inputs:
+      current:
+        description: 'CURRENT "stable" version number (must be published to CDN root)'
+        required: true
+      next:
+        description: 'NEXT version number (must be published to CDN root)'
+        required: true
+
+jobs:
+  pr-dev-build:
+    if: contains(fromJson('["workflow_dispatch"]'), github.event_name)
```

Comment:
[question] If the wf `on` is set to only run on `workflow_dispatch`, is this really needed? I genuinely don't know on this one.

Reply:
nope this is a remnant that can be removed, TY